__tests__/pages/curriculum.test.js

```diff
@@ -39,13 +42,13 @@ describe('Curriculum Page', () => {
       }
     ]
 
-    const { findByRole } = render(
+    const { findByText } = render(
```

Comment:
I know that testing-library recommends using findByRole/getByRole first, but since my tree looks like this only text queries work:
![names](https://user-images.githubusercontent.com/3845012/105536906-2959c980-5d02-11eb-9c01-b6f8276219c4.png)

Reply:
Please add this as a comment to the code. You don't need to include the image but explain the reasoning.

Reply:
Is it really necessary? The code above is rather self explanatory and it's a pretty common pattern in tests (render, query, compare to snapshot). I provided this screenshot to show that I used this query not because it's easier to use but because it's the only way to find correct components. 
If anything I need to learn more about accessibility and change `Error` component to allow proper naming.